mssqlcli/mssqlclioptionsparser.py

```diff
@@ -15,139 +19,139 @@
 def create_parser():
     args_parser = argparse.ArgumentParser(
         prog=u'mssql-cli',
-        description=u'Microsoft SQL Server CLI. ' +
-        u'Version {}'.format(mssqlcli.__version__))
+        description=u'Microsoft SQL Server CLI Version {}'\
+            .format(mssqlcli.__version__))
 
     args_parser.add_argument(
         u'-U', u'--username',
         dest=u'username',
         default=os.environ.get(MSSQL_CLI_USER, None),
         metavar=u'',
-        help=u'Username to connect to the database')
+        help=_(u'Username to connect to the database'))
 
     args_parser.add_argument(
         u'-P', u'--password',
         dest=u'password',
         default=os.environ.get(MSSQL_CLI_PASSWORD, None),
         metavar=u'',
-        help=u'If not supplied, defaults to value in environment variable MSSQL_CLI_PASSWORD.')
+        help=_(u'If not supplied, defaults to value in environment variable MSSQL_CLI_PASSWORD.'))
 
     args_parser.add_argument(
         u'-d', u'--database',
         dest=u'database',
         default=os.environ.get(MSSQL_CLI_DATABASE, None),
         metavar=u'',
-        help=u'database name to connect to.')
+        help=_(u'Database name to connect to'))
```

Comment:
What' happened to the period?

Reply:
Sentence should end with period. Noun should not.

Reply:
Then let's fix the sentence. The convention in this file is to use a period for every help string.

Reply:
> Then let's fix the sentence.

I could not exactly understand what it means. My terminology 'sentence' means a set of words that is complete in itself, typically starts with subject + verb.
* "If not supplied, defaults to value in environment variable MSSQL_CLI_PASSWORD." --> This is a sentence.
* "database name to connect to." --> This is not a sentence, and should not have period at the end as my knowledge of English grammar.

I do not see any consistent convention both in this file and across the project. For example, "Username to connect to the database" does not have period while "database name to connect to." does have. Other string such as "Connection request encountered an exception" in connectionservice.py does not have period.

I rolled back all period change I made in this PR since it seems going out of scope. Let's have the string convention in separate PR later.